python/tvm/relay/op/strategy/cuda.py

```diff
@@ -463,9 +463,9 @@ def batch_matmul_strategy_cuda(attrs, inputs, out_type, target):
     """batch_matmul cuda strategy"""
     strategy = _op.OpStrategy()
     strategy.add_implementation(
-        wrap_compute_batch_matmul(topi.nn.batch_matmul),
+        wrap_compute_batch_matmul(topi.cuda.batch_matmul),
         wrap_topi_schedule(topi.cuda.schedule_batch_matmul),
-        name="batch_matmul.cuda",
+        name="batch_matmul.gpu",
```

Comment:
Why change to `gpu`?

Reply:
I don't insist on it, but we recently (with the great(!) new softmax schedule) had some confusion about cuda schedules being used on non-cuda.

Reply:
Yeah I realized that ROCM uses many CUDA schedules and it results in confusion. We can find lots of CUDA schedules have `if target == 'rocm'`. However, it would be better to have a separate PR to correct this if everone agrees to do so.

Reply:
Yeah, I changed it to CUDA for now.